tests/scalability/run_suite.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+# Fail job when script errors
+set -e
```

Comment:
This doesn't always work, at least not in the way that it's described. 
For example, it does not apply to commands that are in a && or ||  list ([documentation](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html))

To test this, you can run: 
```
#!/usr/bin/env bash

set -e

cd ./nonexistent &&
echo "one"
echo "two"
echo "three"
```

You'll see that `cd ./nonexistent` fails, so `echo "one"` is not executed, but the script does **not** exit and you get both `echo "two"` and `echo "three"` commands outputs. 

I can count at least 3 instances of "&&" in this file in just this small sample view visible in GitHub review UI. 
This risks to become super confusing when someone tries to debug. 

Please remove all "&&" command lists and keep the `set -e` OR add exit code checks for all commands that can fail (which some say is the only correct way to do it).

Reply:
removed those `&&`